<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AbstraktniDialogPrehled</class>
 <widget class="QDialog" name="AbstraktniDialogPrehled">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="sekceVyhledavaciKriterium">
        <property name="title">
         <string>Vyhledávací kritérium:</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout"/>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="sekceCasoveRozmezi">
        <property name="title">
         <string>Časové rozmezí:</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>od:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="vstupDatumOd">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="displayFormat">
            <string>dd. MM. yyyy</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>do:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="vstupDatumDo">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="displayFormat">
            <string>dd. MM. yyyy</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
           <property name="date">
            <date>
             <year>2099</year>
             <month>1</month>
             <day>1</day>
            </date>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>nebo:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxObdobi">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>125</width>
             <height>0</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>(zvolte období)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Den nejbližšího závozu</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Předchozí měsíc</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Předchozí kvartál</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Předchozí rok</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Za celou dobu</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="tlacitkoVyhledat">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Vyhledat (Ctrl+Enter)</string>
           </property>
           <property name="text">
            <string>Vyhledat</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="popisekUpozorneni">
        <property name="styleSheet">
         <string notr="true">color: red;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="tabulka">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="tlacitkoZavrit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Zavřít (Esc)</string>
        </property>
        <property name="text">
         <string>Zavřít</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="tlacitkoExportovatCsv">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Exportovat CSV (Ctrl+E)</string>
        </property>
        <property name="text">
         <string>Exportovat CSV</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>vstupDatumOd</tabstop>
  <tabstop>vstupDatumDo</tabstop>
  <tabstop>comboBoxObdobi</tabstop>
  <tabstop>tlacitkoVyhledat</tabstop>
  <tabstop>tabulka</tabstop>
  <tabstop>tlacitkoZavrit</tabstop>
  <tabstop>tlacitkoExportovatCsv</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
